### DISCUSSION TITLE
Weird behavior of TextField with Codemagic build (Flutter)

### DISCUSSION THREAD

**Discussion by burekas7** [codemagic-ci-cd/codemagic-docs#1405] [Q&A] [answered]
Hi,

I have a flutter app.
When I create a build with Codemagic, the IOS build on TestFlight behaves weird.
Tapping on a TextField widget make the box expands to infinity vertically with grey background.
But when I build it manually with "flutter build ios" and Xcode procedure this problem doesn't exist.

Until two weeks ago everything was fine.

What could be the problem?

Thanks.

**Comment by icarusdust** [comment]
Hi @burekas7, double check your environment variables, if you are using any 3rd party credentials/tokens in your app, try to refresh them.

**Comment by burekas7** [comment]
@icarusdust 

No, I don't have any environment variables or 3rd party credentials/tokens in my app.

The only thing I have in the flow is pre-build script that contains this:
```
rm pubspec.lock
cd ios
rm Podfile.lock
flutter clean
flutter pub get
pod install --repo-update
```

But this script was running before and nothing has been changed.

Also running these command manually and build it manually with "flutter build ios" and xcode validation and distribution, this weird issue doesn't exist.

**Comment by burekas7** [comment]
@icarusdust 
Which flutter version CodeMagic using to build the app?
Maybe this is the difference between my build which is ok and CodeMagic build?

**Comment by burekas7** [comment, marked as answer]
Ok the bug was in my code that raised from flutter 3.7.
I upgraded to 3.7 too and fix the bug.

You can close here.
Thanks.
